plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/nodes/CawsRootNode.kt

```diff
@@ -53,11 +54,16 @@ class CawsRootNode(private val nodeProject: Project) : AbstractTreeNode<String>(
     }
 
     override fun feature() = CodeCatalystConnection.getInstance()
+    companion object {
+        const val CAWS_SIGNED_IN_ACTION_GROUP = "aws.caws.devtools.actions.loggedin"
+        const val CAWS_SIGNED_OUT_ACTION_GROUP = "aws.caws.devtools.actions.loggedout"
+        const val CAWS_EXPIRED_TOKEN_ACTION_GROUP = "aws.caws.devtools.actions.expired"
+    }
 }
 
 class CawsServiceNode : DevToolsServiceNode {
     override val serviceId: String = "aws.toolkit.caws.service"
-    override fun buildServiceRootNode(nodeProject: Project): AbstractTreeNode<*> = CawsRootNode(nodeProject)
+    override fun buildServiceRootNode(project: Project): AbstractTreeNode<*> = CawsRootNode(project)
```

Comment:
```suggestion
    override fun buildServiceRootNode(nodeProject: Project): AbstractTreeNode<*> = CawsRootNode(nodeProject)
```

should not be renaming the parameter unless you're renaming it across all classes